src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/utils/SessionTemplateUtil.kt

```diff
@@ -0,0 +1,60 @@
+package uk.gov.justice.digital.hmpps.visitscheduler.utils
+
+import jakarta.persistence.Tuple
+import org.springframework.stereotype.Component
+import uk.gov.justice.digital.hmpps.visitscheduler.dto.sessions.SessionCapacityDto
+import uk.gov.justice.digital.hmpps.visitscheduler.dto.sessions.category.SessionCategoryGroupDto
+import uk.gov.justice.digital.hmpps.visitscheduler.dto.sessions.incentive.SessionIncentiveLevelGroupDto
+import uk.gov.justice.digital.hmpps.visitscheduler.dto.sessions.location.PermittedSessionLocationDto
+import uk.gov.justice.digital.hmpps.visitscheduler.dto.sessions.location.SessionLocationGroupDto
+import uk.gov.justice.digital.hmpps.visitscheduler.model.entity.session.category.PrisonerCategoryType
+import uk.gov.justice.digital.hmpps.visitscheduler.model.entity.session.incentive.IncentiveLevel
+import java.util.*
+import java.util.stream.Collectors
+
+@Component
+class SessionTemplateUtil {
+  val sessionCapacityComparator: Comparator<SessionCapacityDto> =
+    Comparator { o1: SessionCapacityDto, o2: SessionCapacityDto ->
```

Comment:
What is o1 and o2, please use meaning full names, like 
sessionCapacityOne and sessionCapacityTwo

Reply:
Updated